deploy/pom.xml

```diff
@@ -139,7 +139,7 @@
               <goal>copy-resources</goal>
             </goals>
             <configuration>
-              <outputDirectory>${project.build.directory}/deploy/helm</outputDirectory>
+              <outputDirectory>${project.build.directory}/deploy/helm/eclipse-hono</outputDirectory>
```

Comment:
is there  a particular reason why you want to move this to a subfolder?

Reply:
It is specified by helm that the chart has to be in a folder with the chart name. In fact `helm package` returns an error if the folder is not compliant. There is some [debate on this](https://github.com/helm/helm/issues/1979#issuecomment-280507379) but my understanding is that the rule still stands.